00-INIT-MODULES.sh

```diff
@@ -3,10 +3,11 @@ if [ -z "$MODULEPATH" ]; then
   export MODULEPATH=/etc/modulefiles:/usr/share/modulefiles:/usr/share/Modules/modulefiles:/etc/site/modules
 
   ## Initializations
-  export LMOD_RC=/etc/site/extras/lmodrc.lua
-  ##export LMOD_SYSTEM_NAME=AVX2
-  ##export LMOD_SYSTEM_DEFAULT_MODULES="HPCBIOS/2015q2 sge"
-  export LMOD_SYSTEM_DEFAULT_MODULES="sge cluster"
+  export LMOD_RC=/etc/site/lmod/lmodrc.lua
+  export LMOD_SYSTEM_DEFAULT_MODULES="settarg use.own.eb" ## "HPCBIOS/2015q2 sge cluster"
```

Comment:
just a quick comment, on this and the equivalent `setenv` of `00-INIT-MODULES.sh`, you'd probably like to add:
`dot`, as per `default-environment` modulefile provided by Bright. 

We need to follow up about to organize this optimally, we basically now arrive at the need described in issue #51